crates/nu-command/src/conversions/into/string.rs

```diff
@@ -305,21 +292,31 @@ pub fn action(
         },
     }
 }
-fn format_int(int: i64) -> String {
-    int.to_string()
+fn format_int(int: i64, group_digits: bool, decimals: usize) -> String {
+    let str = if group_digits {
```

Comment:
It seems that `group_digits` doesn't do any useful thing, this is why clippy throw out warning